src/components/ContentNode/CodeListing.vue

```diff
@@ -191,8 +191,11 @@ code {
   // this mask image is not actually used for any visual effect since there is
   // no background being used on this element—however, we need this in order to
   // establish a new stacking context, which resolves a Safari bug where the
-  // scrollbar is not clipped by this element depending on its border-radius
-  -webkit-mask-image: -webkit-radial-gradient(#fff, #000);
+  // scrollbar is not clipped by this element depending on its border-radius.
+  // It's inside a screen query, because it causes issues with printing to PDF.
+  @media screen {
```

Comment:
We should probably also make this change in the `DeclarationSource` component where the same `mask-image` style is being applied.